wdl/SelectBatchesAndColumns.wdl

```diff
@@ -0,0 +1,46 @@
+version 1.0
+
+task SelectBatchesAndColumns {
+	input {
+		File sample_sets_table_tsv
+		Array[String]? batches_opt
+		String column
+		String? docker
+	}
+	
+	Boolean select_batches=defined(batches_opt)
+	Array[String] batches=select_first([batches_opt, ["all"]])
+
+	String docker_set = select_first([docker, "ubuntu:18.04"])
+
+	runtime {
+		docker: docker_set
+	}
+
+	command <<<
+		set -euo pipefail
+		1>&2 echo "Selecting columns from sample sets table"
+		FILE=~{sample_sets_table_tsv}
+
+		COLNAME=~{column}
+		1>&2 echo "COLUMN NAME=$COLNAME"
+
+		COLNUM=$(head -1 $FILE | awk -v RS='\t' -v field="$COLNAME" '$0~field{print NR; exit}')
+		1>&2 echo "COLUMN NUMBER=$COLNUM"
```

Comment:
What happens if the column isn't found? We would want it to catch and exit gracefully with an error message.